sdk/objc/components/video_codec/RTCVideoEncoderH264.mm

```diff
@@ -811,7 +910,12 @@ - (void)frameWasEncoded:(OSStatus)status
     RTC_LOG(LS_ERROR) << "Encode callback failed";
     return;
   }
-  _bitrateAdjuster->Update(frame.buffer.length);
+
+  // if (_encodeMode == Constant) {
```

Comment:
is this because we've deprecated bitrateAdjuster? good to remove then?

Reply:
I tested two days ago only modifying kLimitToAverageBitRateFactor = 1.0f; to
kLimitToAverageBitRateFactor = 5.0f; also seems to get good results (keep bitrateAdjuster). will do a comparison test soon

Reply:
@hiroshihorie , can you try just modifying `kLimitToAverageBitRateFactor = 5.0f` based on the original code?

Reply:
@davidzhao  Yes I think it's ok to remove _bitrateAdjuster.
@cloudwebrtc  kLimitToAverageBitRateFactor 5~8 seems fine but, I think chrome was using 10.0f if my memory was correct. Should we just follow what chrome uses ?

Reply:
> @cloudwebrtc kLimitToAverageBitRateFactor 5~8 seems fine but, I think chrome was using 10.0f if my memory was correct. Should we just follow what chrome uses ?

following the chrome should be fine, and test results are as expected, so great work @hiroshihorie